splunk-otel-android/src/main/java/com/splunk/rum/BackgroundProcessDetector.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.rum;
+
+import android.annotation.SuppressLint;
+import android.app.Application;
+import android.os.Build;
+import java.lang.reflect.Method;
+
+public class BackgroundProcessDetector {
+    public static Boolean isBackgroundProcess(String applicationId) {
+        String applicationProcessName = "";
+        if (Build.VERSION.SDK_INT >= 28) applicationProcessName = Application.getProcessName();
```

Comment:
I'm surprised that this passed spotless checks. We want all `if` statements to be multi-line and have curly brace blocks around them.